Makefile

```diff
@@ -4,7 +4,6 @@ PYTEST := bin/py.test
 GUISRC := jujugui/static/gui/src
 GUIBUILD := jujugui/static/gui/build
 ASSETS_DIR := $(GUISRC)/app/assets
-SVG_SPRITE_SOURCE_DIR := $(GUISRC)/app/assets/svgs
```

Comment:
The `Makefile` isn't being used any longer. I actually left it here just for this one target :) I'll look into whether we can remove it wholesale now.